olive/engine/packaging/sample_code/ONNXModel/cs/code_sample.cs

```diff
@@ -101,19 +117,19 @@ static void Main()
             JObject sessOpts = (JObject)j["session_options"];
 
             // Create inference configuration
-            using InferenceSessionConfiguration options = new InferenceSessionConfiguration();
+            using SessionOptions options = new SessionOptions();
             UpdateSessOptions(options, sessOpts);
             UpdateExecutionProvider(options, executionProvider);
 
             // Create inference session
-            session = new InferenceSession("model.onnx", options);
+            session = new InferenceSession(modelPath, options);
         }
 
         // Run inference
-        // Replace inputNames, inputValues, outputNames with actual variable names
-        using var outputTensors = session.Run(new RunOptions(), inputNames, inputValues, outputNames);
+        // Replace inputs, outputNames with actual variable names
+        using var outputTensors = session.Run(<inputs>, <outputsNames>, new RunOptions());
```

Comment:
better to give some examples so user could easily specify the inputs and outputs

Reply:
and make this file could be successfully build.